ansible/roles/nessus/tasks/main.yml

```diff
@@ -78,28 +78,29 @@
         state: started
   when: registered.rc != 0 or not matching_key|default(false)|bool
 
-# Copy the nessus_base.py to tmp
-- name: Copy the nessus_base python file for importing the base nessus policy
+- name: Create /etc/cyhy directory
+  file:
+    path: /etc/cyhy
+    state: directory
+    owner: cyhy
+    group: cyhy
+    mode: 0750
+
+- name: Create the configuration file for Nessus API access
+  template:
+    src: nessus_api.yml.j2
+    dest: /etc/cyhy/nessus_api.yml
+    owner: cyhy
+    group: cyhy
+    mode: 0640
```

Comment:
Alphabetize?
```suggestion
  file:
    group: cyhy
    mode: 0750
    owner: cyhy
    path: /etc/cyhy
    state: directory

- name: Create the configuration file for Nessus API access
  template:
    dest: /etc/cyhy/nessus_api.yml
    group: cyhy
    mode: 0640
    owner: cyhy
    src: nessus_api.yml.j2
```

Reply:
I intentionally did not alphabetize to match the existing Ansible role structures and I was planning to update #478 once this PR is merged which would have those changes.

Reply:
OK, I suspected you had some motive here, but I wasn't sure what it was.  Thanks for the explanation. 👍

Reply:
Just to circle back I have rebased #478 and the concerns raised here should be resolved in https://github.com/cisagov/cyhy_amis/pull/478/commits/904f7e1c522dd5d7fe6460fec5b3b77d5c8e24ce.